libs/telemetry/src/main/java/org/opensearch/telemetry/metrics/MetricsRegistry.java

```diff
@@ -54,13 +53,12 @@ public interface MetricsRegistry extends Closeable {
      * Creates the Observable Gauge type of Metric. Where the value provider will be called at a certain frequency
      * to capture the value.
      *
-     * @param name          name of the observable gauge.
-     * @param description   any description about the metric.
-     * @param unit          unit of the metric.
-     * @param valueProvider value provider.
-     * @param tags          attributes/dimensions of the metric.
+     * @param name        name of the observable gauge.
+     * @param description any description about the metric.
+     * @param unit        unit of the metric.
+     * @param value       value provider.
      * @return closeable to dispose/close the Gauge metric.
      */
-    Closeable createGauge(String name, String description, String unit, Supplier<Double> valueProvider, Tags tags);
```

Comment:
I think we could keep this method in case tags do not change per metric?

Reply:
I agree, we can keep both. Let me do the change.